scripts/linux.sh

```diff
@@ -22,12 +22,12 @@ for disk in $DISKS; do
   echo "mountpoint: $mountpoint"
   echo "name: $device"
 
-  # We determine if a drive is a system drive
-  # by checking the removeable flag.
-  # There might be a better way in GNU/Linux systems.
-  removable=`lsblk -d $device --output RM | ignore_first_line | trim`
-
-  if [[ "$removable" == "1" ]]; then
+  eval "`udevadm info --query=property --export --export-prefix=UDEV_ --name=$disk`"
+  if [[ "`lsblk -d $device --output RM | ignore_first_line | trim`" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_DRIVE_MEDIA_FLASH_SD" == "1" ]] || \
+     [[ "$UDEV_ID_BUS" == "usb" ]]
```

Comment:
Uh. With this explanation this makes sense. You can either rename it or state this in a comment or so.